### PR TITLE
Merge pull request #273 from keboola/beta

### PR DESCRIPTION
**Pull request by MonikaFeigler**
MAIN: breaking changes

### CODE REVIEW

File: .keboola/repository.json [hunk 1]
```diff
@@ -437,7 +422,6 @@
             "keboola.ex-google-analytics-v4",
             "keboola.wr-google-sheets",
             "keboola.wr-google-bigquery-v2",
-            "keboola.wr-db-snowflake",
             "keboola.wr-snowflake-blob-storage",
```
Comment (MonikaFeigler) [inline .keboola/repository.json:425]: AWS support ico for blob storage, so I deleted wr.db.snowflake so it's not visible twice there

**Comment by MonikaFeigler** [comment]
Big Query data destination was added

**Comment by MonikaFeigler** [comment]
plus an condition for 1 table in the output was added

**Comment by MonikaFeigler** [comment]
plus instance id was added to metas of configs

**Comment by MonikaFeigler** [comment]
hi @ujovlado can you please approve? There are minor changes. 
- updated names of configs
- new bigquery data destination for page engagement template
- small update of icos within templates

Thank you.

**Review by michaljurecko — APPROVED** [review]
LGTM

**Pull request merged**